streamer/src/nonblocking/quic.rs

```diff
@@ -34,6 +34,8 @@ use {
 const QUIC_TOTAL_STAKED_CONCURRENT_STREAMS: f64 = 100_000f64;
 const WAIT_FOR_STREAM_TIMEOUT_MS: u64 = 1;
 
+pub const ALPN_TPU_PROTOCOL_ID: &[u8] = b"solana-tpu";
```

Comment:
QUIC may be used for other, non-TPU protocols in the future. do we get to just make these strings up at will or should we use something more generic from the start

Reply:
@t-nelson We can make them up. The IANA keeps a list of IDs [here](https://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml#alpn-protocol-ids) and we can send a request according to [RFC 8447](https://www.rfc-editor.org/rfc/rfc8447.html). Since the identifier targets a "single application"and contains a brand name, I wouldn't bother.

Reply:
@t-nelson Is it okay to keep this constant hardcoded here or should I make it a function argument that's passed down from client/server creation?